xls/dslx/stdlib/std.x

```diff
@@ -396,6 +396,41 @@ fn round_up_to_nearest_test() {
     assert_eq(u32:4, round_up_to_nearest(u32:4, u32:2));
 }
 
+// Returns `x` rounded up to `y`, where `y` is a positive power of 2
```

Comment:
curious why `y` is necessary? wouldn't a general "next power of two" rounding be useful as it? (similar to https://en.cppreference.com/w/cpp/numeric/bit_ceil in c++)

Reply:
I needed this particular version of a function in calculations of offsets in the ZSTD history buffer. I hope, that soon we will open a public PR that will justify the usage of that function or the function proposed in https://github.com/google/xls/pull/1238#discussion_r1439347935.